ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-ospf.j2

```diff
@@ -25,6 +25,9 @@ router ospf {{ process_id }}
 {%         if router_ospf.process_ids[process_id].max_lsa is defined %}
    max-lsa {{ router_ospf.process_ids[process_id].max_lsa }}
 {%         endif %}
+{%         if router_ospf.process_ids[process_id].default_information_originate is defined and router_ospf.process_ids[process_id].default_information_originate == true %}
```

Comment:
The name for the key 'default_information_originate' is misleading as the configuration generated has the option 'always'. This section should be refactored with the possibility to add more options in the future (like metrics, metric-types and route-maps) and have 'always' as an option as well.

Reply:
Fixed